src/bundle/Resources/public/scss/_general.scss

```diff
@@ -25,6 +25,9 @@ body {
 
 .ez-side-menu,
 .ez-context-menu {
+    .ez-button-container {
+        position: fixed;
```

Comment:
Are you sure it's cross-browser safe? We're supporting the latest browser versions.
Also, this change will affect the right side menu in the location view, not only in the edit view. Is it desired?
What if there are many buttons? What's your experience with a such case.
Also, to improve browser performance you might want to add a property `will-change: transform`.

Reply:
I actually switched to `position:sticky`, which won't lock the container to the initial position, rather it will stick to a fixed position from top only when scrolling. I also added your suggestion for `will-change`. As for support for this feature, it is in newest browser versions and I don't think we need a browser polyfill since this is only an enhancement.

While I looked at this briefly today, I noticed there is actually an issue with the content type selector, etc. widgets (see [video](https://youtu.be/mFSTTP6DqkA)), which will also need to be handled for this to be an improvement over current functionality.